Corona-Warn-App/src/main/java/de/rki/coronawarnapp/task/TaskController.kt

```diff
@@ -137,7 +136,19 @@ class TaskController @Inject constructor(
             this.putAll(it)
         }
 
-        Timber.tag(TAG).v("Tasks after processing: %s", this.values)
+        if (size > TASK_HISTORY_LIMIT) {
+            Timber.v("Enforcing history limits (%d), need to remove %d.", TASK_HISTORY_LIMIT, size - TASK_HISTORY_LIMIT)
+            values
+                .filter { it.isFinished }
+                .sortedBy { it.startedAt }
```

Comment:
minor: with a history of 50 probably just an edge case but wouldn't be sorted by finishedAt be more beneficial? In case we have long and short running tasks?

Reply:
Good point. Done.